root/utility/getSortName.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import ENTITIES from '../../entities';
+
+/*
+ * Returns the sort name for entities that have them, or falls back to
+ * the name otherwise.
+ */
+
+export default function getSortName(entity: CoreEntityT) {
+  const hasSortName = ENTITIES[entity.entityType].sort_name;
+  return hasSortName && entity.sort_name
```

Comment:
The entity.sort_name check seems to be needed to calm Flow down.

Reply:
This might be useful too: https://flow.org/en/docs/types/comments/#toc-comment-type-include
```suggestion
  return hasSortName /*:: && entity.sort_name */
```
Flow will treat the comment as if it were code, but it won't actually be executed. Sometimes useful for nudging the type checker like here while avoiding unnecessary checks at run time.